test/fixtures/thermostats.yml

```diff
@@ -4,8 +4,9 @@
 # model remove the '{}' from the fixture names and add the columns immediately
 # below each fixture, per the syntax in the comments below
 #
-one: {}
+thermostat_one: {}
 # column: value
 #
-two: {}
+thermostat_two:
```

Comment:
The rims_id definitely inserts as expected. The rest of it I expect to be handled by the controller. But for some reason its missing something along the way. Both RIMS and thermostat have the same issue. Similar explicit id setup worked for rhizome/particle device before I did it with the automated association.